### PR TITLE
add VBNT-K MST firmware, add VBNT-L rbi, add VBNT-S firmwares, add and fix for VBNT-F firmwares

### PR DESCRIPTION
**Pull request by FrancYescO**

### CODE REVIEW

File: docs/Repository.md [hunk 1]
```diff
@@ -197,7 +197,8 @@ A basic ADSL only BCM6362 based gateway. Very useful as SIP ATA.
 
 | Type   | Version          | Timestamp  | Root Strategy | Mirror |
 |:------:|:-----------------|:-----------|:--------------|:-------|
-| 2      | 16.3.7190        | 2016-10-06 | #A            | [HTTPS](https://github.com/hack-technicolor/hack-technicolor/raw/master/firmware/myrepublic-vbnt-l_16.3.7190-2761005-bank_dump.xz) - *note: this **IS NOT** an RBI firmware, it is a raw bank dump, you can't use with TFTP or regular firmware upgrade tools* |
+| 2      | 16.3.7190        | 2016-10-06 | #A            | [HTTPS](https://github.com/hack-technicolor/hack-technicolor/raw/master/firmware/myrepublic-vbnt-l_16.3.7190-2761005-bank_dump.xz?raw=true) - *note: this **IS NOT** an RBI firmware, it is a raw bank dump, you can't use with TFTP or regular firmware upgrade tools* |
```
Comment (LuKePicci) [inline docs/Repository.md:200]: Should be either:
`/blob/...?raw=true`
or
`/raw/...`

File: docs/Repository.md [hunk 2]
```diff
@@ -309,6 +317,12 @@ A basic ADSL only BCM6362 based gateway. Very useful as SIP ATA.
 
 ## DGA4130 / VBNT-K
 
+### MST
+
+| Type   | Version   | Timestamp  | Root Strategy | Mirror |
+|:------:|:----------|:-----------|:--------------|:-------|
+| ???    | 17.3.0165 | 2018-02-28 | -             | [HTTPS](https://github.com/hack-technicolor/hack-technicolor/raw/master/firmware/MST%20DGA4130%2017.3.0165.1681007.rbi?raw=true) |
```
Comment (LuKePicci) [inline docs/Repository.md:324]: Same blob vs raw dilemma

Comment (LuKePicci) [reply docs/Repository.md:324]: Isn't this Type 2 for strategy #D?

**Review by LuKePicci — CHANGES_REQUESTED** [review]

File: docs/Repository.md (hunk 1 above)
Comment (LuKePicci) [reply docs/Repository.md:200]: maybe one redirects to the other or something like that

File: docs/Repository.md (hunk 2 above)
Comment (FrancYescO) [reply docs/Repository.md]: idk for strategy

Comment (LuKePicci) [reply docs/Repository.md]: #D is the same as for UNO's vant-6, did anybody test it on this firmware?

**Review by LuKePicci — APPROVED** [review]

**Pull request merged**